From f8819ebb82c2bc76236b1d56267df41ca4bd9254 Mon Sep 17 00:00:00 2001
From: Kevin Minehart <kmineh0151@gmail.com>
Date: Thu, 15 Jun 2023 15:12:11 -0500
Subject: [PATCH] Add Sub function and test

---
 main.go      | 5 +++++
 main_test.go | 6 ++++++
 2 files changed, 11 insertions(+)

diff --git a/main.go b/main.go
index c34396b..451dad2 100644
--- a/main.go
+++ b/main.go
@@ -6,6 +6,11 @@ func Add(a, b int) int {
 	return a + b
 }
 
+func Sub(a, b int) int {
+	return a - b
+}
+
 func main() {
 	log.Println(Add(1, 2))
+	log.Println(Sub(2, 1))
 }
diff --git a/main_test.go b/main_test.go
index fc0509f..33c8e8b 100644
--- a/main_test.go
+++ b/main_test.go
@@ -10,3 +10,9 @@ func TestAdd(t *testing.T) {
 		t.Fatalf("Add(2, 2) should equal '%d' but equals '%d'", 3, res)
 	}
 }
+
+func TestSub(t *testing.T) {
+	if res := Sub(2, 1); res != 1 {
+		t.Fatalf("Sub(2, 1) should equal '%d' but equals '%d'", 1, res)
+	}
+}
-- 
2.41.0

